kurl_util/cmd/bashmerge/main.go

```diff
@@ -145,15 +145,21 @@ func parseBashFlags(installer *kurlv1beta1.Installer, bashFlags string) error {
 			}
 			installer.Spec.Kubernetes.LoadBalancerAddress = split[1]
 		case "ekco-enable-internal-load-balancer":
-			if installer.Spec.Ekco != nil {
-				installer.Spec.Ekco.EnableInternalLoadBalancer = true
+			if installer.Spec.Ekco == nil {
+				installer.Spec.Ekco = &kurlv1beta1.Ekco{}
+			}
+			installer.Spec.Ekco.EnableInternalLoadBalancer = true
```

Comment:
does this have an effect if there's no ekco version supplied?

I don't like changing this unless ekco is no longer required

Reply:
changed back